Library/Homebrew/cmd/test-bot.rb

```diff
@@ -425,6 +425,10 @@ def formula formula_name
       install_args = %w[--verbose]
       install_args << "--build-bottle" unless ARGV.include? "--no-bottle"
       install_args << "--HEAD" if ARGV.include? "--HEAD"
+      # Pass --devel or --HEAD to the test-bot in the event formulae lack stable. Supports devel-only/head-only.
+      # head-only should not have devel, but devel-only can have head. Stable can have all three.
+      install_args << "--devel" if formula.devel && !formula.stable?
+      install_args << "--HEAD" if formula.head && !formula.stable? && !formula.devel?
```

Comment:
Would probably rather see this maybe skip to the `formula.devel` part below instead? Out of interest, what's happening with head-only PRs as I thought they were working OK (I could be wrong)?